Purchase/index.ts

```diff
@@ -28,23 +29,29 @@ export namespace Purchase {
 			isoly.DateTime.is(value.modified) &&
 			Payment.is(value.payment) &&
 			(typeof value.amount == "undefined" || Amount.is(value.amount)) &&
-			(Receipt.is(value.receipt) ||
-				(typeof value.receipt == "object" && typeof value.receipt.to == "string") ||
-				typeof value.receipt == "undefined")
+			typeof value.email == "string" &&
+			Array.isArray(value.receipt) &&
+			value.receipt.every((receipt: unknown) => typeof receipt == "object" && receipt && Receipt.is(receipt))
 		)
 	}
 	export function create(
 		purchase: Purchase.Creatable,
 		token: string,
+		organizationId: string,
+		to: string,
 		idLength: cryptly.Identifier.Length = 8
 	): Purchase {
 		const now = isoly.DateTime.now()
+		const id = cryptly.Identifier.generate(idLength)
+		const [recipient, domain] = to.split("@")
 		return {
-			id: cryptly.Identifier.generate(idLength),
+			id: id,
 			created: now,
 			modified: now,
 			...purchase,
 			payment: Payment.create(purchase.payment, token),
+			email: `${recipient}+${organizationId}|${id}@${domain}`,
```

Comment:
do we want to do this at this stage? currently we can only recieve receipts at receipt@issuefab.com